fern/docs.yml

```diff
@@ -74,206 +177,22 @@ navigation:
                 path: ./docs/pages/overview/define-your-api/ferndef/api-yml.mdx
               - page: Packages
                 path: ./docs/pages/overview/define-your-api/ferndef/packages.mdx
-              - page: Comparison with OpenAPI
-                path: ./docs/pages/overview/define-your-api/ferndef/comparison.mdx
-      - section: Fern CLI tool
-        slug: cli
-        contents:
-          - page: Overview
-            path: ./docs/pages/overview/cli/cli-overview.mdx
-          - section: CLI commands
-            slug: commands
-            contents:
-              - page: fern generate
-                path: ./docs/pages/overview/cli/cli-commands/fern-generate.mdx
-              - page: fern check
-                path: ./docs/pages/overview/cli/cli-commands/fern-check.mdx
-              - page: fern add
-                path: ./docs/pages/overview/cli/cli-commands/fern-add.mdx
-              - page: fern init
-                path: ./docs/pages/overview/cli/cli-commands/fern-init.mdx
-              - page: fern write-definition
-                path: ./docs/pages/overview/cli/cli-commands/fern-write-definition.mdx
-              - page: fern upgrade
-                path: ./docs/pages/overview/cli/cli-commands/fern-upgrade.mdx
-              - page: fern token
-                path: ./docs/pages/overview/cli/cli-commands/fern-token.mdx
-              - page: fern login
-                path: ./docs/pages/overview/cli/cli-commands/fern-login.mdx
-              - page: fern format
-                path: ./docs/pages/overview/cli/cli-commands/fern-format.mdx
-              - page: fern help
-                path: ./docs/pages/overview/cli/cli-commands/fern-help.mdx
-              - page: fern version
-                path: ./docs/pages/overview/cli/cli-commands/fern-version.mdx
-      - section: Changelog
-        contents:
-          - page: December 2023
-            path: ./docs/pages/overview/changelog/december-2023.mdx
-          - page: February 2024
-            path: ./docs/pages/overview/changelog/february-2024.mdx
-  - tab: sdks
-    layout:
-      - page: Overview
-        path: ./docs/pages/fern-sdks/sdks-overview.mdx
-      - section: Fern SDKs Quickstarts
-        slug: quickstarts
-        contents:
-          - page: Fern SDKs Quickstart with OpenAPI
-            slug: openapi
-            path: ./docs/pages/fern-sdks/sdks-quickstarts/sdks-quickstart-openapi.mdx
-          - page: Fern SDKs Quickstart with Fern Def
-            slug: ferndef
-            path: ./docs/pages/fern-sdks/sdks-quickstarts/sdks-quickstart-ferndef.mdx
-      - section: SDK generators
+      - section: Fern CLI
         contents:
-          - page: TypeScript SDK
```

Comment:
Why remove the generator-specific pages? There is generator-specific content, such as the run times that the TS SDK is compatible with.

Reply:
I was going back and forth on this and I think either we crate pages for each language, and then list features on each page. Or pages for each featurs, and show examples for each language. Ultimately opted for the second because I think it's more intuitive. 

In the `overview` page for SDKs, planning is to create a card for every generator and provide a sentence. I think we should title the TypeScript generator `Universal TypeScript SDK generator` or something.

Reply:
I think the benefit of `per feature` is that we get to advertise the functionality. Big pro